sdk/src/logs/readable_log_record.cc

```diff
@@ -0,0 +1,51 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#ifdef ENABLE_LOGS_PREVIEW
+
+#  include <cstddef>
+#  include <type_traits>
+
+#  include "opentelemetry/sdk/logs/readable_log_record.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace logs
+{
+
+nostd::string_view ReadableLogRecord::GetSeverityText() const noexcept
+{
+  std::size_t severity_index = static_cast<std::size_t>(GetSeverity());
+  if (severity_index >= std::extent<decltype(opentelemetry::logs::SeverityNumToText)>::value)
+  {
+    return opentelemetry::logs::SeverityNumToText[0];
+  }
+
+  return opentelemetry::logs::SeverityNumToText[severity_index];
+}
+
+const opentelemetry::sdk::instrumentationscope::InstrumentationScope &
+ReadableLogRecord::GetDefaultInstrumentationScope() noexcept
+{
+  // FIXME: Use shared default instrumentation scope?
```

Comment:
nit - is this comment valid, as we seems to be using shared static instrumentation scope below.

Reply:
The shared default instrumentation scope I mean is a shared instrumentation scope between all of the components of SDK, includes trace and metrics, not just for logs.